browsers/edge.json

```diff
@@ -192,10 +192,10 @@
         },
         "99": {
           "release_date": "2022-03-03",
-          "release_notes": "https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-archive-stable-channel#version-990115030-march-3",
           "status": "retired",
           "engine": "Blink",
-          "engine_version": "99"
+          "engine_version": "99",
+          "release_notes": "https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-archive-stable-channel#version-990115030-march-3"
```

Comment:
Sorry about the unrelated change here. I used a semi-automated script to do these PRs, and for some reasons, it sometimes does this.
I can revert that specific change if you think that's better.

Reply:
These changes are totally fine -- I know that since release notes get moved to an "archive" the links change, so this helps make sure all the links still work when that happens.  👍